### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by wejendorp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
+    "eslint-plugin-standard": ">=2.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Tradeshift/eslint-config-tradeshift.git"
+  },
+  "scripts": {
+    "test": "tape test/*.js"
+  }
 }
```
Comment (sqren) [inline package.json]: meh. What if I don't use `np`? Will it then revert to tabs?

**Comment by sqren** [comment]
👾

File: package.json (hunk 1 above)
Comment (wejendorp) [inline package.json]: I was just testing it, will change this commit to be version only. Sorry

**Comment by wejendorp** [comment]
This has been pushed, travis just needs a beating.

**Comment by sqren** [comment]
image

**Comment by sqren** [comment]
What happened to the beating?

**Pull request merged**